### PR TITLE
PWA Manifest file section, with manifest and iconography

### PR DESCRIPTION
**Pull request by estelle**
files for part 3: the manifest section of the cycletracker app tutorial

Note that all the files are in all the directories so people can jump in at any point in the tutorial. The new content is:

* the manifest JSON file.
* the icons
* the two lines in the header of the HTML doc (lines 7-8)
* the return false line which should have been in the javascript_functionality section, so it's added to both that file and the manifest directory copy. (line 39 in app.js)

The following content can be reviewed, but doesn't need to be as it has already been approved and merged:

* the CSS file `styles.css`.
* the rest of the HTML `index.html`, other than the lines  7-8 in the `<head>`
* the rest of the JS `app.js` file, other than the `return false` ( line 39)

### CODE REVIEW

File: cycletracker/manifest_file/cycletracker.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/* cycletracker.json */
```
Comment (wbamberg) [inline cycletracker/manifest_file/cycletracker.json]: JSON doesn't support comments.

File: cycletracker/manifest_file/cycletracker.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+{
+  "name": "cycleTracker: Period Tracking app",
+  "short_name": "CT",
+  "description": "Securily and confidentially track your menstrual cycle. Enter the start and end dates of your periods, saving your private data to your browser on your device, without sharing it with the rest of the world.",
```
Comment (wbamberg) [inline cycletracker/manifest_file/cycletracker.json]:
```suggestion
  "description": "Securely and confidentially track your menstrual cycle. Enter the start and end dates of your periods, saving your private data to your browser on your device, without sharing it with the rest of the world.",
```

**Review by wbamberg — COMMENTED** [review]
I didn't look at the JS or CSS, since it has already been reviewed.

**Review by wbamberg — APPROVED** [review]
👍

**Review by achtenmichael960-star — APPROVED** [review]

**Pull request merged**